unity-client/Assets/Scripts/MainScripts/DCL/Components/Billboard/Billboard.cs

```diff
@@ -33,6 +33,12 @@ public override IEnumerator ApplyChanges(string newJson)
             {
                 //NOTE(Zak): We have to wait one frame because if not the entity will be null. (I'm Brian, but Zak wrote the code so read this in his voice)
                 yield return null;
+
+                if (entity == null)
+                {
+                    Debug.LogWarning("It seems skipping a frame didnt work, entity is still null");
+                    yield break;
+                }
```

Comment:
maybe, it could be a good idea to use a `while` here and retry like for 5 frames? just in case? what do you think?

Reply:
I would attack that stuff in the #1467 and just leave this PR for the null-checks.